CMakeLists.txt

```diff
@@ -23,7 +23,7 @@
 
 cmake_minimum_required (VERSION 3.11 FATAL_ERROR)
 
-find_package(IRODS 4.3.3 EXACT REQUIRED CONFIG)
+find_package(IRODS 4.3.4 EXACT REQUIRED CONFIG)
```

Comment:
Is this not already updated to 4.3.4 on the main branch?

Reply:
Looks like it. This diff will disappear once this is rebased, I think.